configs/world.conf

```diff
@@ -196,7 +200,8 @@
 #        Default: 0 (off)
 #
 
-<Server PlayerLimit          = "100"
+<Server CacheVersion         = "12340"
```

Comment:
CacheVersion is equal ro GameBuild.... why do we need 2 different configs with the same values?